plugins/hybrid_analysis/plugin.spec.yaml

```diff
@@ -1,145 +1,1109 @@
 plugin_spec_version: v2
 extension: plugin
-products: [insightconnect]
+products: [ insightconnect ]
 name: hybrid_analysis
 title: Hybrid Analysis
 vendor: rapid7
 support: community
-status: []
+status: [ ]
 description: Lookup file hashes to determine if they are malicious
-version: 2.0.2
+version: 3.0.0
 supported_versions:
-  - Hybrid Analysis API v1
+  - Hybrid Analysis API v2
 resources:
   source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/hybrid_analysis
   license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
   vendor_url: https://www.hybrid-analysis.com/
 tags:
-- malware
-- sandbox
-- malware analysis
+  - malware
+  - sandbox
+  - malware analysis
 hub_tags:
-  use_cases: [threat_detection_and_response]
-  keywords: [malware_analysis]
-  features: []
+  use_cases: [ threat_detection_and_response ]
+  keywords: [ malware_analysis ]
+  features: [ ]
 types:
-  report:
-    environment_id:
+  certificates:
+    issuer:
+      title: Issuer
+      type: string
+      description: Issuer
```

Comment:
Descriptions shouldn't match the title exactly, could this be updated? 

A similar pattern throughout the plugin.spec.yaml. file.

First word in the description should have the first letter capitalized and not be same as title (e.g. Match string from text file).

Reply:
@ablakley-r7 According to the style guide, different descriptions are used in actions and triggers, not in custom output types.